code/__DEFINES/jobs.dm

```diff
@@ -182,3 +182,11 @@ GLOBAL_LIST_INIT(jobs_fallen_marine, typecacheof(list(/datum/job/fallen/marine),
 #define ENGINEER_TOTAL_BUY_POINTS 75
 /// How many points the field commander can spend
 #define COMMANDER_TOTAL_BUY_POINTS 45
+
+GLOBAL_LIST_INIT(default_marine_points, list(
+		"SMARTGUNNER SUPPLIES" = DEFAULT_TOTAL_BUY_POINTS,
```

Comment:
Please use the defines instead of strings for the `"SMARTGUNNER SUPPLIES"` (swap with `CAT_SGSUP`, etc.) since that's what `marine_selector_cats` uses.

Reply:
I tried that and something goes wrong when I do that

Reply:
Just tried it on local, it works fine unless I missed something. 

```
GLOBAL_LIST_INIT(default_marine_points, list(
		CAT_SGSUP = DEFAULT_TOTAL_BUY_POINTS,
        CAT_ENGSUP = ENGINEER_TOTAL_BUY_POINTS,
		CAT_LEDSUP = DEFAULT_TOTAL_BUY_POINTS,
		CAT_MEDSUP = MEDIC_TOTAL_BUY_POINTS,
		CAT_FCSUP = COMMANDER_TOTAL_BUY_POINTS,
	))
```